src/NuGet.Services.SearchService/NuGet.Services.SearchService.csproj

```diff
@@ -88,7 +88,7 @@
       <Version>2.8.1</Version>
```

Comment:
What about this TraceListener version?

Reply:
I didn't touch that one and left it as-is everywhere. This is the package that caused us perf issues in the past.